File: src/TKDHubAPI.Domain/Entities/ClassSchedule.cs
namespace TKDHubAPI.Domain.Entities;

public partial class ClassSchedule : BaseEntity
{
    public int TrainingClassId { get; set; }
    public TrainingClass TrainingClass { get; set; }
    public DayOfWeek Day { get; set; }
    public TimeOnly StartTime { get; set; }
    public TimeOnly EndTime { get; set; }
}

File: Frontend/tkd_hub_web/src/app/types/ClassSchedule.ts
import { DayOfWeek } from "./DayOfWeek";

export type ClassSchedule = {
  id?: number | null;
  day: DayOfWeek; // Use the enum for day
  startTime: string; // Start time in "HH:mm" format
  endTime: string; // End time in "HH:mm" format
};
